libbeat/docker-compose.yml

```diff
@@ -67,13 +24,8 @@ services:
       test: ["CMD-SHELL", "curl -u admin:testing -s http://localhost:9200/_cat/health?h=status | grep -q green"]
       retries: 300
       interval: 1s
-
-  elasticsearch_monitoring:
```

Comment:
The tests for this have been disabled for 2 years, there's no reason to keep starting it.